template/deploy/helm/[[product]]-operator/templates/serviceaccount.yaml.j2

```diff
@@ -0,0 +1,75 @@
+# =============
+# This file is automatically generated from the templates in stackabletech/operator-templating
+# DON'T MANUALLY EDIT THIS FILE
+# =============
+{{- if .Values.serviceAccount.create -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "operator.serviceAccountName" . }}
+  labels:
+    {{- include "operator.labels" . | nindent 4 }}
+  {{- with .Values.serviceAccount.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  # "namespace" omitted since ClusterRoles are not namespaced
+  name: {{ include "operator.serviceAccountName" . }}-role
+  labels:
+    {{- include "operator.labels" . | nindent 4 }}
+rules:
```

Comment:
The rules are operator-specific and should be configurable.

see [rules for the Superset operator](https://github.com/stackabletech/superset-operator/blob/4892ca21162ccf187ace4b5b276fb17a9f1a2a06/deploy/role/superset-cluster-role.yaml)